litellm/utils.py

```diff
@@ -1222,7 +1222,7 @@ def litellm_telemetry(data):
         payload = {
             "uuid": uuid_value,
             "data": data,
-            "version": pkg_resources.get_distribution("litellm").version,
+            "version:": importlib.metadata.version("litellm"),
```

Comment:
can we confirm this works ?

Reply:
Yeah, I'll do a video of building -> importing a fresh build -> calling

Reply:
The video with full cycle showing it works for me: https://app.dadan.io/video/share/y4wOwHDV0RwNP26f